packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/simulate-ensemble-ciemss-operation.ts

```diff
@@ -20,12 +20,23 @@ export interface SimulateEnsembleWeight {
 	value: number;
 }
 
+export const speedValues = Object.freeze({
+	numSamples: 1,
+	method: CiemssMethodOptions.euler
+});
```

Comment:
Euler-based methods usually means you need to have some notion of step-size, can we check if that is needed here?

Reply:
I will need to add the step-size 
I will be updating pyciemss-service shortly for my next PR so I was planning on doing that there if thats alright
`update pyciemss and EnsembleSimulationCiemssRequest to not take a timespan but just an endTime.`

Reply:
Sure

Reply:
Thanks